data/core/units/dwarves/Miner.cfg

```diff
@@ -6,15 +6,19 @@
     image=units/dwarves/miner.png
     hitpoints=25
     movement_type=dwarvishfoot
+    [resistance]
+        blade=90
+        pierce=90
+        impact=90
+    [/resistance]
     movement=4
-    level=1
+    level=0
     alignment=neutral
-    advances_to=null
-    {AMLA_DEFAULT}
-    experience=50
-    cost=5
+    advances_to=Dwarvish Fighter,Dwarvish Guardsman,Dwarvish Ulfserker
+    experience=19
```

Comment:
change this to 24 
(from Hejnewar, the Mp dev whose current github account has been locked until further notice).

Reply:
Done.